compiler/x/env/OMRCPU.cpp

```diff
@@ -116,39 +116,3 @@ OMR::X86::CPU::testOSForSSESupport(TR::Compilation *comp)
    {
    return false;
    }
-
-bool
-OMR::X86::CPU::getX86OSSupportsSSE(TR::Compilation *comp)
-   {
-   // If the FXSR (bit 24) and SSE (bit 25) bits in the feature flags are not set, SSE is unavailable
-   // on this processor.
-   //
-   uint32_t flags = self()->getX86ProcessorFeatureFlags(comp);
-
-   if ((flags & 0x03000000) != 0x03000000)
-      return false;
-
-   return self()->testOSForSSESupport(comp);
-   }
-
-bool
-OMR::X86::CPU::getX86OSSupportsSSE2(TR::Compilation *comp)
-   {
-   // If the FXSR (bit 24) and SSE2 (bit 26) bits in the feature flags are not set, SSE is unavailable
-   // on this processor.
-   //
-   uint32_t flags = self()->getX86ProcessorFeatureFlags(comp);
-   if ((flags & 0x05000000) != 0x05000000)
-      return false;
-
-   return self()->testOSForSSESupport(comp);
-   }
-
-bool
-OMR::X86::CPU::getX86SupportsTM(TR::Compilation *comp)
```

Comment:
Just curious why this is being deleted because its orthogonal to the purpose of this PR.

I realize it's unused and likely could be done better so I don't have any objections.  However,  if you want to roll this deletion into this PR then please amend your commit message to broaden the scope of this PR.

Reply:
Thank-you for pointing it out.
Yes, they are being deleted because they are redundant and unused. I've updated the commit message.